zerver/tests/test_message_edit.py

```diff
@@ -1708,6 +1710,140 @@ def notify(user_id: int) -> Dict[str, Any]:
             )
         )
 
+    @mock.patch("zerver.actions.user_topics.send_event")
+    def test_edit_unmuted_topic(self, mock_send_event: mock.MagicMock) -> None:
+        stream_name = "Stream 123"
+        stream = self.make_stream(stream_name)
+        hamlet = self.example_user("hamlet")
+        cordelia = self.example_user("cordelia")
+        othello = self.example_user("othello")
+        aaron = self.example_user("aaron")
+        self.subscribe(hamlet, stream_name)
+        self.login_user(hamlet)
+        message_id = self.send_stream_message(
+            hamlet, stream_name, topic_name="Topic1", content="Hello World"
+        )
+
+        self.subscribe(cordelia, stream_name)
+        self.login_user(cordelia)
+        self.subscribe(othello, stream_name)
+        self.login_user(othello)
+        self.subscribe(aaron, stream_name)
+        self.login_user(aaron)
```

Comment:
It would be pretty useful to have a high-level comment explaining the setup here that can be used to verify the assertions at the end are as expected more quickly.

Reply:
[added comment(s)](https://github.com/zulip/zulip/pull/24742/commits/7c0d033369fab5f965d4538894a328f106f08329#diff-3a0986e298aade053cf2a8360758507bc4fed8c3d1fe341724e6c7fd0dd720a6R1602)